Apps/W1/Shopify/app/src/Companies/Pages/ShpfyCompLocations.Page.al

```diff
@@ -0,0 +1,82 @@
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Sales.Customer;
+
+/// <summary>
+/// Page Shpfy Company Locations (ID 30165).
+/// </summary>
+page 30165 "Shpfy Comp. Locations"
+{
+    ApplicationArea = All;
+    Caption = 'Shopify Company Locations';
+    Editable = false;
+    InsertAllowed = false;
+    DeleteAllowed = false;
+    ModifyAllowed = false;
+    PageType = List;
+    SourceTable = "Shpfy Company Location";
+    UsageCategory = None;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(General)
+            {
+                field(Id; Rec.Id)
+                {
+                    ToolTip = 'Specifies the unique identifier for the company location in Shopify.';
```

Comment:
Ahh, the table was already there. I'd still consider moving the tooltip to the table, as this is the first Page, for the table...

Reply:
I would like to have tooltips on table level, but I don't know... 
There are no tables with Tooltips in whole Shopify connector solution.
It would be a bit strange for me to have one object developed in different way.